modules/ROOT/pages/enhanced-skins-and-icon-packs.adoc

```diff
@@ -1,79 +1,99 @@
+NOTE: Enhanced skins are only available with a link:{pricingpage}/[paid {productname} subscription].
 
-== How to use a Enhanced skins
+The {prem_skins_icons} provide pre-made skins and icon packs to quickly customize {productname}'s appearance. These premium options allow integrators to match {productname} to their application's design system.
 
-NOTE: Enhanced skins are only available with a link:{pricingpage}/[paid {productname} subscriptions].
+== Available premium skins
 
-Use the xref:editor-skin.adoc#skin[skin] option, in combination with the xref:add-css-options.adoc#content_css[content_css] option and the values listed below.
+The following premium skins are available for {productname}:
 
-Available values for xref:editor-skin.adoc#skin[skins]:
+* `material-classic`
+* `material-outline`
+* `bootstrap`
+* `fabric`
+* `fluent`
+* `borderless`
+* `small`
+* `jam`
+* `naked`
+* `outside`
+* `snow`
 
-* material-classic
-* material-outline
-* bootstrap
-* fabric
-* fluent
-* borderless
-* small
-* jam
-* naked
-* outside
-* snow
+For information on configuring skins, see: xref:editor-skin.adoc[Skin options].
 
-Available values for xref:add-css-options.adoc#content_css[content_css]:
+Some skins include both UI styles (`+skin.css+`) and content styles (`+content.css+`). When a skin includes both files, using the `+skin+` option will apply both UI and content styles. The `+content_css+` option is not required in this case.
 
-* material-classic
-* material-outline
-* fabric
-* fluent
+For styling only the editable area (without changing the UI), separate content-only CSS files are available. These files are located in the content folder and may differ from the content styles included with skins. Available content-only options:
 
-=== Example: using a Enhanced skin
+* `material-classic`
+* `material-outline`
+* `fabric`
+* `fluent`
+
+Use the xref:add-css-options.adoc#content_css[`+content_css+`] option to apply content-only styles.
+
+Skins can include both UI styles (`+skin.css+`) and content styles (`+content.css+`). Using the `+skin+` option applies both automatically when both files are present.
+
+For styling only the editable area (without changing the UI), separate content-only CSS files are available. These files are located in the content folder and may differ from the content styles included with skins. Available content-only options:
+
+* `material-classic`
+* `material-outline`
+* `fabric`
+* `fluent`
+
+Use the xref:add-css-options.adoc#content_css[`+content_css+`] option to apply content-only styles.
+
+=== Example: using a premium skin
 
 [source,js]
 ----
 tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
-  skin: 'fabric',        // Name of the skin
-  content_css: 'fabric'  // Name of the content skin
+  selector: 'textarea',
+  skin: 'fabric'  // Applies both UI and content styles (if both files are included)
```

Comment:
Should we note this requires skin to be in the default location, and link to the page with skin and skin_url?